Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -470,22 +478,25 @@ def search_records(
         search_field_name = ''
         search_field_id = ''
         fields_mapping = level_data['mapping']
+        level_id = level_data['level']
```

Comment:
can you elaborate on this field? will it always appear in the level data?

Reply:
We add it in the code [here](https://github.com/demisto/content/blob/master/Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py#L354)